circle/functions

```diff
@@ -128,6 +129,43 @@ install_google_cloud_sdk() {
   fi
 }
 
+install_azure_cli() {
+  if ! which az >/dev/null ; then
+    info "Installing Python3 and its dependencies..."
+    apk update
+    if ! which python >/dev/null; then
+      apk add --no-cache build-base libffi-dev openssl-dev python3 python3-dev
+      python3 -m ensurepip
+      rm -r /usr/lib/python*/ensurepip
```

Comment:
What does this remove do? And the previous command?

Reply:
Solution based on https://hub.docker.com/r/frolvlad/alpine-python2/~/dockerfile/